OWML.ModLoader/ModData.cs

```diff
@@ -66,15 +68,16 @@ private void MakeConfigConsistentWithDefault()
             {
                 if (!IsSettingSameType(Config.Settings[key], DefaultConfig.Settings[key]))
                 {
-                    TryUpdate(key, Config.Settings[key], DefaultConfig.Settings[key]);
+                    wasCompatible &= TryUpdate(key, Config.Settings[key], DefaultConfig.Settings[key]);
```

Comment:
i've never seen `&=` before :P 
same as `wasCompatible = wasCompatible && whatever`?

Reply:
actually, yeah, using bitwise on boolean might be iffy